APIGOV-11241 Update CLI create environment documentation

rsavajiyani opened the pull request:
@markbice As a part of this story(https://techweb.axway.com/jira/browse/APIGOV-11241) I updated the CLI docs to include sample resource files users can use to create environments using the CLI. I'd appreciate your feedback.

@andreamussap Thank you for your help with my last documentation story; I'd appreciate any input you have regarding the updates I've made as a part of this pull request.

andreamussap commented on content/en/docs/central/cli_environments.md:
```diff
@@ -2,11 +2,11 @@
-*Estimated reading time*: 1 minute
+*Estimated reading time*: 5 minutes
 
 {{% alert title="Note" %}}This feature is currently in development and will be available soon.{{% /alert %}}
```
@rsavajiyani do you mean that this feature is not yet available for production use? In this case, for consistency, can you please make this note the same as [Add your hybrid environment](https://axway-open-docs.netlify.com/docs/central/add_hybrid_env/)

{{< alert title="Public beta" color="warning" >}}This feature is currently in **public beta** and not yet available for production use.{{< /alert >}}

andreamussap commented:
@rsavajiyani  I've made some updates.

I'd like you to add a short introduction/description from sections' List environments' onward. One short sentence is enough, we just can't start them with 'examples', it's missing some information about what the section is about.

**Direct preview link** - https://deploy-preview-202--axway-open-docs.netlify.com/docs/central/cli_environments/

I can have a look at the content again after @markbice  provides his feedback.

rsavajiyani commented:
> @rsavajiyani I've made some updates.
> 
> I'd like you to add a short introduction/description from sections' List environments' onward. One short sentence is enough, we just can't start them with 'examples', it's missing some information about what the section is about.
> 
> **Direct preview link** - https://deploy-preview-202--axway-open-docs.netlify.com/docs/central/cli_environments/
> 
> I can have a look at the content again after @markbice provides his feedback.

@andreamussap Thanks for your feedback. I've added some short introductory text before the environment samples; let me know if you think it's appropriate. @markbice / @robertengaxway I'd appreciate your feedback as well.

markbice commented on content/en/docs/central/cli_environments.md:
```diff
@@ -27,48 +27,769 @@ Learn how to create and manage your distributed cloud and on-premise environment
-## List environments
+### Other examples of valid resource types
+
+Use and reference the following examples of valid resource types to practice creating multiple resources.
```
Perhaps we should clarify this are all based on the "create env" by saying something like: Use and reference the following json examples of multiple additional resource types possible to configure for an environment.

rsavajiyani replied:
Thanks for the feedback Mark, I'll implement your suggestion.

markbice commented:
I like it, with the one change (in comment above) I noted. 
Pls doublecheck if the additional examples should be the "cleaner" ones you would use on the **create** and not these more verbose ones you would typically see on a **get**

andreamussap commented:
@rsavajiyani TBH I'm more confused now: section 'other examples of valid resource types' - These are valid resource types for what, to create environments? It seems so to me. 

The problem is that your text (You can use to create multiple resources) doesn't match mbice suggestion (multiple additional resource types to configure for an environment)!?? What are these examples for after all ?

Pleas rename the title to 'Other valid resource types' (remove word 'examples' - it's stated in the description already) and the clarify the description (e.g. 'You can use the following json examples of multiple additional resource types to... create/ configure?? an environment)

Also pls see @markbice comment 'additional examples should be the "cleaner" ones ...not these more verbose ones'. I don't know what he means so Mark will have to review this when you're finished.

rsavajiyani commented:
> @rsavajiyani TBH I'm more confused now: section 'other examples of valid resource types' - These are valid resource types for what, to create environments? It seems so to me.
> 
> The problem is that your text (You can use to create multiple resources) doesn't match mbice suggestion (multiple additional resource types to configure for an environment)!?? What are these examples for after all ?
> 
> Pleas rename the title to 'Other valid resource types' (remove word 'examples' - it's stated in the description already) and the clarify the description (e.g. 'You can use the following json examples of multiple additional resource types to... create/ configure?? an environment)
> 
> Also pls see @markbice comment 'additional examples should be the "cleaner" ones ...not these more verbose ones'. I don't know what he means so Mark will have to review this when you're finished.

Thanks for the feedback Andrea. I've used Mark's suggested wording as it's more clear, and I've renamed the title of the section to exclude the word "examples".

rsavajiyani commented:
> I like it, with the one change (in comment above) I noted.
> Pls doublecheck if the additional examples should be the "cleaner" ones you would use on the **create** and not these more verbose ones you would typically see on a **get**

Thanks Mark, I've cleaned up the json examples and removed the unnecessary key value pairs that aren't needed for the "create" action.

andreamussap commented:
@markbice are you happy with the additional examples now? Can I merge these changes to production?

andreamussap commented:
### Thank you
We really appreciate your contribution and thank you for helping us to improve this documentation!
 
### Where to find the updated documentation
An Axway technical writer has reviewed and merged your PR, which is now available on the following sites:
Netlify (staging): https://axway-open-docs.netlify.com/docs/
Zoomin (production): https://docs.axway.com/bundle/axway-open-docs/
 
### Send us your feedback
If you have a moment, please complete the following 2 minute survey to give us your feedback on this process and help us make it even better:
https://www.surveymonkey.com/r/PJ5HQFD
